packages/select/src/components/query-list/queryList.tsx

```diff
@@ -40,6 +41,32 @@ export interface IListItemsProps<T> extends IProps {
      */
     itemRenderer: ItemRenderer<T>;
 
+    /**
+     * Custom renderer for the contents of the dropdown.
+     *
+     * The default implementation invokes `itemRenderer` for each item that passes the predicate
+     * and wraps them all in a `Menu` element. If the query is empty then `initialContent` is returned,
+     * and if all items are filtered away then `noResults` is returned.
```

Comment:
Nit: "if there are no items that match the predicate" (`items` could be empty, for all we know.)

Reply:
_nice_